fdbserver/workloads/AtomicOps.actor.cpp

```diff
@@ -48,8 +48,7 @@ struct AtomicOpsWorkload : TestWorkload {
 		apiVersion500 = ((sharedRandomNumber % 10) == 0);
 		TraceEvent("AtomicOpsApiVersion500").detail("ApiVersion500", apiVersion500);
 
-		lbsum = 0;
-		ubsum = 0;
+		logsum = 0;
```

Comment:
Did you decide that having the upper bound is not sufficiently useful to track it?

Reply:
Yes. Having it is to check the txn_unkown error.